src/colorizer/ColorizeCanvas.ts

```diff
@@ -152,17 +154,15 @@ export default class ColorizeCanvas {
   private renderer: WebGLRenderer;
   private pickRenderTarget: WebGLRenderTarget;
 
-  protected dataset: Dataset | null;
-  protected track: Track | null;
-  protected canvasResolution: Vector2 | null;
-
-  protected featureKey: string | null;
-  protected selectedBackdropKey: string | null;
-  protected colorRamp: ColorRamp;
-  protected colorMapRangeMin: number;
-  protected colorMapRangeMax: number;
+  protected params: RenderCanvasStateParams | null;
```

Comment:
Replaced a lot of the tracked properties on the left with a single object, `params`.